example.php

```diff
@@ -4,4 +4,5 @@
 // is_null is a function that does stuff 
```

Comment:
@CodelingoBotTester rewrite line "// This is an entirely uninitiated interactive rewrite"

Reply:
@CodelingoBotTester rewrite line "// This is an entirely uninitiated interactive rewrite"

Reply:
Success! Committed rewritten file.